src/views/list_view.rs

```diff
@@ -281,60 +325,45 @@ impl View for ListView {
         let offset = self.labels_width() + 1;
 
         debug!("Offset: {}", offset);
+        let mut yOffset = 0;
         for (i, child) in self.children.iter().enumerate() {
             match child {
                 ListChild::Row(ref label, ref view) => {
-                    printer.print((0, i), label);
+                    printer.print((0, yOffset), label);
                     view.draw(
-                        &printer.offset((offset, i)).focused(i == self.focus),
+                        &printer.offset((offset, yOffset)).focused(i == self.focus),
                     );
+                    yOffset += view.important_area(child.size).height();
```

Comment:
This is the line posing problem, important_area is not the correct function to call to get the proper height.